src/back/services/update.ts

```diff
@@ -9,25 +9,78 @@ import { UpdateAttributes, UpdateStatus } from '../../entities/Updates/types'
 import { getCurrentUpdate, getNextPendingUpdate, getUpdateUrl } from '../../entities/Updates/utils'
 import { inBackground } from '../../helpers'
 import { DiscourseService } from '../../services/DiscourseService'
+import { ErrorService } from '../../services/ErrorService'
+import { FinancialService } from '../../services/FinancialService'
+import { ErrorCategory } from '../../utils/errorCategories'
 
 import { CoauthorService } from './coauthor'
 import { DiscordService } from './discord'
 import { EventsService } from './events'
 
 export class UpdateService {
   static async getById(id: UpdateAttributes['id']) {
-    return await UpdateModel.findOne<UpdateAttributes>({ id })
+    try {
+      const update = await UpdateModel.findOne<UpdateAttributes>({ id })
+      if (update) {
+        const records = await FinancialService.getRecords(update.id)
+        if (records) {
+          return { ...update, records }
```

Comment:
How about calling this update variable `financialRecords` instead of `records`? 
Outside of the financial service the word record can be more ambiguous